Add doc for pulsar_publish_rate_limit_times (#13958)
apache/pulsar: site2/docs/reference-metrics.md

| pulsar_consumers_count | Gauge | The number of active consumers of the topic connected to this broker. |
| pulsar_rate_in | Gauge | The total message rate of the topic coming into this broker (messages/second). |
| pulsar_rate_out | Gauge | The total message rate of the topic going out from this broker (messages/second). |

> | pulsar_publish_rate_limit_times | Gauge | The number of times the publishing rate limit was triggered (times/minute). |

Reviewer: ```suggestion
| pulsar_publish_rate_limit_times | Gauge | The number of times the publish rate limit is triggered in one minute. |
```
do you mean this?

P.S. Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. Details: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.e8uqh1awkcnp
